src/macaron/slsa_analyzer/registry.py

```diff
@@ -564,5 +752,31 @@ def _should_skip_check(check: BaseCheck, results: dict[str, CheckResult]) -> Ski
 
         return None
 
+    # TODO: Come up with a better name for this method.
+    def get_check_tree(self) -> CheckTree:
```

Comment:
This method traverse the check tree as a whole without any include/exclude restrictions. Therefore, we could print out the whole run checks tree with all registered checks in the final HTML report.

Reply:
- [ ] I think an example of the returned dictionary of this method would be useful.

Reply:
Added in https://github.com/oracle/macaron/pull/254/commits/f6d7426d7b9f937477fbe5ab5cda809edd6dc18b